WebRoot/js/ajax/dwt/widgets/DwtMenu.js

```diff
@@ -923,7 +923,7 @@ function(which, preventFocus) {
 	while (currItem) {
 		if (!currItem.isStyle) { // this is not a DwtMenuItem
 			if (!preventFocus) {
-				currItem.focus();
+				setTimeout(function() {currItem.focus();}, 1);
```

Comment:
Put a comment on why we are doing this in code

Reply:
Added.